src/libraries/System.Security.Cryptography/tests/MLDsaOpenSslTests.Unix.cs

```diff
@@ -116,11 +116,11 @@ public void MLDsaOpenSsl_DuplicateKeyHandleLifetime()
 
         private static void VerifyInstanceIsUsable(MLDsaOpenSsl mldsa)
         {
-            byte[] seed = new byte[mldsa.Algorithm.PrivateSeedSizeInBytes];
-            Assert.Equal(mldsa.Algorithm.PrivateSeedSizeInBytes, mldsa.ExportMLDsaPrivateSeed(seed)); // does not throw
+            byte[] seed = mldsa.ExportMLDsaPrivateSeed();
+            Assert.Equal(mldsa.Algorithm.PrivateSeedSizeInBytes, seed.Length); // does not throw
 
-            byte[] secretKey = new byte[mldsa.Algorithm.SecretKeySizeInBytes];
-            Assert.Equal(mldsa.Algorithm.SecretKeySizeInBytes, mldsa.ExportMLDsaSecretKey(secretKey)); // does not throw
+            byte[] secretKey = mldsa.ExportMLDsaSecretKey();
+            Assert.Equal(mldsa.Algorithm.SecretKeySizeInBytes, secretKey.Length); // does not throw
 
```

Comment:
The "Does not throw" comments feel misplaced to me.  Assuming they are commenting the line that they are the end of, the only possible "throw" would be that `seed.Length` threw ANE.

They should either be somewhere that makes sense, or just deleted.